common/source/docs/common-mamba405-mk2.rst

```diff
@@ -0,0 +1,112 @@
+================================
+Mamba F405 MK2 Flight Controller
+================================
+
+.. image:: ../../../images/mamba_f405mk2_board.jpg
+     :target: ../_images/mamba_f405mk2_board.jpg
+
+https://www.diatone.us/products/mamba-f405-flight-controller-mk2
+
+The Mamba F405 MK2 is a flight controller produced by
+`Diatone <https://www.diatone.us>`__.
+
+.. note::
+
+   Support for this board is available with Plane 4.1.0, and higher
+
+Features
+========
+
+-  STM32F405RGT6 microcontroller
+-  MPU6000 IMU
+-  AT7456E OSD
+-  4 UARTs
+-  4 PWM outputs
+
+Pinout
+======
+
+.. figure:: ../../../images/mamba_f405mk2_pinout.png
+   :target: ../_images/mamba_f405mk2_pinout.png
+
+   Mamba F405 MK2 Board
+
+UART Mapping
+============
+
+======= ================ ================
+Name    Pin              Function
+======= ================ ================
+SERIAL0 COMPUTER         USB
+SERIAL1 PPM/RX1/SBUS/TX1 UART1 (RC Input)
```

Comment:
Using RX1 as UART is not possible unless alt config is selected....see how this is done here: https://ardupilot.org/copter/docs/common-omnibusf4pro.html

Reply:
Looking at this closer, I'm unclear on the author's assertions of SERIALx usage. Do you know if there are compelling considerations to list GPS on `SERIAL6` instead of the more common `SERIAL3`, for instance?